stdlib/kokorawr/kokorawr/functions/services/scores.js

```diff
@@ -8,7 +8,12 @@ const lib = require('lib')({token: process.env.STDLIB_TOKEN});
 */
 module.exports = (post=false, store={}, context, callback) => {
```

Comment:
Look at the Rock Paper Scissors score

```
$ lib .commands.scores bob dude
{
  "response_type": "in_channel",
  "text": "Hi <@bob>! Here are the current scores:\nTic Tac Toe: Red(0) Blue(0)\nConnect 4: Red(0) Blue(0)\nBattleship: Red(0) Blue(0)\nDuck Duck Goose: Red(0) Blue(0)\nRock Paper Scissors: Red(0) Blue(0)\nMastermind: Red(0) Blue(0)\n\nBoth teams are tied at (NaN:NaN)"
}
```

Reply:
what should the rps score be?

Reply:
EDIT: I meant the thing that was tied at the end. RPS score is okay @anatasiavela